types/address.go

```diff
@@ -17,6 +17,18 @@ import (
 )
 
 const (
+	// Constants defined here are the defaults value for address.
+	// You can use the specific values for your project.
+	// Add the follow lines to the `main()` of your server.
+	//
+	//	config := sdk.GetConfig()
```

Comment:
Should probably keep the indenting consistent 👍

Reply:
Yes, it should be. IDE insert `space` instead of `tab`